### PR TITLE
Register prototype updates before round 2 of research

### PR DESCRIPTION
**Pull request by jesseyuen**
Another screenshot dump and some high level notes on significant user-facing updates to the prototype. 

I pledge a promise to the design history gods that moving forward smaller, incremental entries will be made that cover more of the _why_ about our updates.

### CODE REVIEW

File: app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+* The 'Recommend for QTS' flow has been renamed to 'Record QTS outcome' and updated to acknowledge that a trainee record will not always be recommended for QTS, and instead may be required to record a 'not passed' outcome.   
+* A [prototype settings](https://register-prototype.herokuapp.com/admin) page has been added which enables features to be toggled on and off. 
+* Ongoing iteration of content throughout the prototype with an focus on ensuring our use of terminology is consistent.
+* We have hidden the guidance section to test the hypothesis that guidance may not be needed.
+
+## Screenshots of all pages
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{{ appScreenshots({
+  hideContents: true,
```
Comment (fofr) [inline app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md]: Why are you hiding the contents? The contents are there to make scanning long lists of screenshots more easy.

**Review by fofr — DISMISSED** [review]

File: app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md (hunk 1 above)
Comment (jesseyuen) [reply app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md]: Good shout @fofr  🦅👀

File: app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      img: { src: "2020.11.02_12_25_02_Add a trainee - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add undergraduate degree",
+      img: { src: "2020.11.02_12_25_11_Add undergraduate degree - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add undergraduate degree",
+      img: { src: "2020.11.02_12_25_18_Add undergraduate degree - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add undergraduate degree",
+      img: { src: "2020.11.02_12_25_26_Add undergraduate degree - Register trainee teachers - GOV.UK.png" }
```
Comment (fofr) [inline app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md]: The title of each screenshot should be unique

Comment (jesseyuen) [reply app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md]: Sorted.

**Review by fofr — APPROVED** [review]

**Pull request merged**